packages/xs-extra-dummy/xs-toolstack.master/opam

```diff
@@ -8,6 +8,7 @@ authors:        "Christian Lindig <christian.lindig@citrix.com>"
 homepage:       "https://github.com/xapi-project/xs-opam"
 license:        "BSD"
 depends: [
+  "angstrom"
```

Comment:
Could we instead depend on the `xapi-clusterd` package in `xs-extra-dummy`?

Reply:
It is already there, maybe we can add all the dummy packages as dependency? @gaborigloi  says they are installable because they have no build rule, just purely dependencies.
These are the dummy packages: rrdd-plugins, xapi-clusterd-test, xapi-clusterd, xenctrl

Reply:
No - this packages has no URL because it is private and can't be installed (I believe).

Reply:
I have tried installing it with `opam install xapi-clusterd` (with opam 2) and it worked for me.

Reply:
https://github.com/xapi-project/xs-opam/blob/master/packages/xs-extra-dummy/xapi-clusterd.master/opam

Reply:
We've added a dummy opam package for each of these, containing just the dependencies, and we deleted the other fields.

Reply:
I have added more dummy packages.